easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -526,11 +526,17 @@ def test_step(self):
             if self.testcmd:
                 testcmd = self.testcmd % {'python': self.python_cmd}
                 cmd = ' '.join([extrapath, self.cfg['pretestopts'], testcmd, self.cfg['testopts']])
-                run_cmd(cmd, log_all=True, simple=True)
+                if return_testcmd_output:
+                    testcmd_output, testcmd_exit_code = run_cmd(cmd, log_all=True, simple=False)
+                else:
+                    run_cmd(cmd, log_all=True, simple=True)
```

Comment:
You can simplify this safely to just:

```python
testcmd_output, testcmd_exit_code = run_cmd(cmd, log_all=True, simple=False)
```

That way you'll always catch the test output/exit code, yet only actually return it if `return_testcmd_output` is `True`.

I think it would even be safe to simply *always* return.

Since nothing is returned now (which implies returning `None`), you can't really break something by making that change (unless weird things are being done).